frontend/src/components/institute/AuthoritiesList.vue

```diff
@@ -132,8 +132,8 @@
                 id="user-search-button"
                 class="ml-3"
                 text="Search"
-                @click-action="searchButtonClick"
                 :disabled="!searchEnabled()"
+                @click-action="searchButtonClick"
               />
             </v-col>
           </v-row>
```

Comment:
This page will likely also have a footer-props that can be cleaned up.

Also, the itemsPerPageOptions can be deleted from both pages if it isn't used anywhere else

Reply:
Resolved